Playbooks/Create-Zendesk-Ticket/azuredeploy.json

```diff
@@ -12,7 +13,7 @@
             "tier": "community" 
         },
         "author": {
-            "name": "Lior Tamir"
+            "name": "Brian Delaney"
```

Comment:
Take the comment to an email.

Reply:
Saw the email. As I understand we leave it as is then, right?